ppl/zqd/handlers_test.go

```diff
@@ -453,6 +447,8 @@ func TestPostZngLogs(t *testing.T) {
 	}, info)
 }
 
+// Skipped trying to convert this one to ZSON for now.
+// See https://github.com/brimsec/zq/issues/2057#issuecomment-803187964
```

Comment:
Just adding a comment to make sure this stands out in the sea of changes.

Reply:
I think leaving it is fine since this test is really about feeding the log-post logic text that doesn't work with the auto-detector so no need to change the buggy tzng to buggy zson.

Reply:
I added a comment about the funky zson enum error to #2057